go.mod

```diff
@@ -62,7 +62,7 @@ require (
 	github.com/hashicorp/golang-lru v0.5.4
 	github.com/hashicorp/hcl v1.0.0
 	github.com/hashicorp/hcp-scada-provider v0.2.3
-	github.com/hashicorp/hcp-sdk-go v0.46.1-0.20230519164650-51657675d9e7
+	github.com/hashicorp/hcp-sdk-go v0.48.0
```

Comment:
Update to released version of HCP SDK, the previous version was commit hash as there was no release triggered.